cmd/admxgen/defs/mount.yaml

```diff
@@ -20,9 +20,39 @@
 
     The supported protocols are the same as the ones supported by gvfs.
     They are listed on the man page of gvfs, under the gvfs-backends section: https://manpages.ubuntu.com/manpages/jammy/en/man7/gvfs.7.html
+    It's up to the user to ensure that the requested protocols are valid and supported and that the shared directories have the correct configuration for the requested connection.
     
   elementtype: "multiText"
   release: "any"
   type: "mount"
   meta:
     strategy: "append"
+
+- key: "/system-mounts"
+  displayname: "System mounts"
+  explaintext: |
+    Define network shares that will be mounted for the system.
+    If more shares are defined higher in the GPO hierarchy, the entries listed here will be appended to the list and duplicates will be removed.
+    
+    Values should be in the format: 
+        <protocol>://<hostname-or-ip>/<shared-dir>
+    e.g.
+        nfs://example_nfs.com/nfs_shared_dir
+        smb://example_smb.com/smb_shared_dir
+        ftp://ftp_share_server.com
+
+    This pattern must be followed, otherwise the policy will not be applied.
+    
+    By default, the mounts will be done in anonymous mode. In case of authentication needed, a krb5 tag can be added to the value, e.g.
+        [krb5]<protocol>://<hostname-or-ip>/<shared-dir>
+
+    If the tag is added, the mount will require Kerberos authentication in order to occur.
+
+    The supported protocols / filsystems are the same as the ones supported by the mount command.
```

Comment:
You shold mention the systemd mount unit doc there.

Reply:
Well, the [systemd.mount]{https://www.freedesktop.org/software/systemd/man/systemd.mount.html} doc is kind of lackluster. It just tells you about the syntax of the mount section, basically. Information about the mount protocols and mount options is given on the [mount man page]{https://manpages.ubuntu.com/manpages/jammy/en/man8/mount.8.html} and the systemd.mount actually points you towards it, that's why I decided to reference directly the mount page, since the user does not need to worry about the unit syntax.

Reply:
ack then :)